tools/gfx/d3d12/render-d3d12.cpp

```diff
@@ -3512,6 +3512,13 @@ class D3D12Device : public RendererBase
 #endif
 
         virtual SLANG_NO_THROW void SLANG_MCALL close() override { m_cmdList->Close(); }
+
+        virtual SLANG_NO_THROW Result SLANG_MCALL
+            getNativeHandle(NativeHandle* outHandle) override
+        {
+            *outHandle = (uint64_t)m_cmdList.readRef();
```

Comment:
You probly want get() instead of readRef(). 
readRef returns a pointer to the CommandList pointer.